src/app/(main)/about/page.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";
+import styled from "styled-components";
+import Image from "next/image";
+import { darkTheme } from "@kleros/ui-components-library";
+import KlerosLogo from "@/assets/kleros-pink.svg";
+import LightLinkButton from "@/components/LightLinkButton";
+
+export default function About() {
+  const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+    padding: 20px 20px 0 20px;
+    text-align: center;
+    gap: 26px;
+  `;
+
+  const Heading = styled.h2`
+    font-weight: 400;
+  `;
+
+  const StyledH2 = styled.h2`
+    font-weight: 400;
+    color: ${darkTheme.klerosUIComponentsPrimaryBlue};
+  `;
+
+  const StyledText = styled.p`
+    font-weight: 200;
+  `;
+
+  const StyledLinkButton = styled(LightLinkButton)`
+    width: 100%;
+  `;
+
+  const LogoContainer = styled.div`
+    display: "flex";
+    justifycontent: "center";
+  `;
+
```

Comment:
Styled components should go outside of the component.

```suggestion

const Container = styled.div`
  display: flex;
  flex-direction: column;
  padding: 20px 20px 0 20px;
  text-align: center;
  gap: 26px;
`;

const Heading = styled.h2`
  font-weight: 400;
`;

const StyledH2 = styled.h2`
  font-weight: 400;
  color: ${darkTheme.klerosUIComponentsPrimaryBlue};
`;

const StyledText = styled.p`
  font-weight: 200;
`;

const StyledLinkButton = styled(LightLinkButton)`
  width: 100%;
`;

const LogoContainer = styled.div`
  display: "flex";
  justifycontent: "center";
`;

export default function About() {
```